docs/contributing/using-an-online-dev-environment.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Using an Online Dev Environment
```

Comment:
I would mention GitPod specifically here as this is GitPod specific documentation